src/app/beer-edit/beer-edit-form/beer-edit-form.component.html

```diff
@@ -0,0 +1,41 @@
+<mat-card>
+    <mat-card-content>
+        <form class="beer-edit-form" [formGroup]="beerForm">
+            <mat-form-field class="full-width">
+                <input matInput placeholder="Beer Name" name="name" formControlName="name">
+            </mat-form-field>
+            <mat-form-field class="full-width">
+                <input matInput placeholder="Beer Style" name="style" formControlName="style">
+            </mat-form-field>
+            <mat-form-field class="full-width">
+                <textarea matInput placeholder="Beer Description" name="description" formControlName="description"></textarea>
+            </mat-form-field>
+            <mat-form-field class="full-width">
+                <textarea matInput placeholder="Beer Ingredients" name="ingredients" formControlName="ingredients"></textarea>
+            </mat-form-field>
+            <mat-grid-list cols="2" rowHeight="100px" gutterSize="20px">
+                <mat-grid-tile>
+                    <mat-form-field class="full-width">
+                        <input type="number" matInput placeholder="ABV" name="abv" formControlName="abv">
+                    </mat-form-field>
+                </mat-grid-tile>
+                <mat-grid-tile>
+                    <mat-form-field class="full-width">
+                        <input type="number" matInput placeholder="EPM" name="epm" formControlName="epm">
+                    </mat-form-field>
+                </mat-grid-tile>
+            </mat-grid-list>
+            <mat-form-field class="full-width">
+                <textarea matInput placeholder="Brewery" name="brewery" formControlName="brewery"></textarea>
+            </mat-form-field>
+        </form>
+    </mat-card-content>
+    <mat-card-actions>
+        <button mat-button (click)="goBack()">BACK</button>
+        <button mat-raised-button onclick="document.getElementById('fileToUpload').click()">
```

Comment:
I am not sure why but file dialog opens twice

Reply:
does it? that's weird... maybe a Chrome thing? I didn't test it on Chrome...